Infrastructure/src/main/java/common/parsers/JsonParser.java

```diff
@@ -0,0 +1,51 @@
+package common.parsers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+
+public class JsonParser {
+
+    private static ObjectMapper mapper = new ObjectMapper();
```

Comment:
Notice you still have other places in your code using `ObjectMapper`. Find and delete them